.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+## Description
+
+Clearly and concisely describe the issue or suggestion here:
+
+
+
+
+
+
+
+### Checklist
+
+*Put an `x` in the boxes that apply. You can also fill these out after creating the issue.*
```

Comment:
Probably want to say 'Change the space in the box to an 'x' for those that apply', as I constantly get people doing `[x ]` on the CSFG.